test/JsonRpc.Tests/InputHandlerTests.cs

```diff
@@ -128,8 +128,10 @@ public void ShouldPassInUtf8EncodedRequests()
         }
 
         [Theory]
-        [InlineData("{\"changes\": [{\"uri\": \"file:///Mörkö.cs\",\"type\": 1}]}")]
-        [InlineData("{\"textDocument\": {\"uri\": \"file://abc/123/树.cs\"}}")]
+        // Mörkö
```

Comment:
I used to be really afraid of Mörkö as a kid

Reply:
Who hasn't been afraid of Mörkö? 😅

Reply:
rofl